packages/deploy-configurations/configs/base.conf.ts

```diff
@@ -0,0 +1,1211 @@
+import { loadContractNames } from '@deploy-configurations/constants'
+import { SystemConfig } from '@deploy-configurations/types/deployment-config'
+import { Network } from '@deploy-configurations/types/network'
+
+const SERVICE_REGISTRY_NAMES = loadContractNames(Network.BASE)
+
+export const config: SystemConfig = {
+  mpa: {
+    core: {
+      ServiceRegistry: {
+        name: 'ServiceRegistry',
+        deploy: false,
+        address: '0x0c1EDa5544EA63cf3d365912343161913a8f19Eb',
+        history: [],
+        constructorArgs: [ 0 ]
+      },
+      OperationExecutor: {
+        name: 'OperationExecutor',
+        deploy: true,
+        address: '0xb65Cbf9689979e1749B38Ba5D5155c70500292D8',
+        serviceRegistryName: SERVICE_REGISTRY_NAMES.common.OPERATION_EXECUTOR,
+        history: [],
+        constructorArgs: [ 'address:ServiceRegistry' ]
+      },
+      OperationStorage: {
+        name: 'OperationStorage',
+        deploy: true,
+        address: '0x2Cb950c62Bc2e76DAEb86223E5F5D46EE9048EA4',
+        serviceRegistryName: SERVICE_REGISTRY_NAMES.common.OPERATION_STORAGE,
+        history: [],
+        constructorArgs: [ 'address:ServiceRegistry', 'address:OperationExecutor' ]
+      },
+      OperationsRegistry: {
+        name: 'OperationsRegistry',
+        deploy: true,
+        address: '0x1F15c0832bF01094C077A780ea85dC7Cfe6C209C',
+        serviceRegistryName: SERVICE_REGISTRY_NAMES.common.OPERATIONS_REGISTRY,
+        history: [],
+        constructorArgs: []
+      },
+      DSProxyFactory: {
+        name: 'DSProxyFactory',
+        deploy: true,
+        address: '0x6eB5e634773E271F7FA7222e4244438Ea74674F1',
+        serviceRegistryName: SERVICE_REGISTRY_NAMES.common.DS_PROXY_FACTORY,
+        history: [],
+        constructorArgs: []
+      },
+      DSProxyRegistry: {
+        name: 'DSProxyRegistry',
+        deploy: true,
+        address: '0x29CCc2C12054Ae0A5eE6FE5120784D33ec17B7E9',
+        serviceRegistryName: SERVICE_REGISTRY_NAMES.common.DS_PROXY_REGISTRY,
+        history: [],
+        constructorArgs: [ 'address:DSProxyFactory' ]
+      },
+      DSGuardFactory: {
+        name: 'DSGuardFactory',
+        deploy: true,
+        address: '0x89010CBE61Ba3A800aFDc20D22E8163c6256E5Ef',
+        serviceRegistryName: SERVICE_REGISTRY_NAMES.common.DS_GUARD_FACTORY,
+        history: [],
+        constructorArgs: []
+      },
+      AccountGuard: {
```

Comment:
Do we need those maker contracts?

Reply:
Probably we don't need it. But it's deployed already. We could remove it from config for the future (in all networks or on L2s only)